_posts/2023-07-15-interview-platform-insights-and-learnings.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title: "Interview-platform: Insights and Learnings"
+tldr: rajat
+modified: 2023-07-15 21:05:38 +0530
+category: technology
+tags: [video-conferencing,socket-io,webrtc,mediasoup]
+author: 
+image:
+  feature: 
+  credit: 
+  creditlink: 
+comments: 
+share: 
+---
+
+**INTERVIEW PLATFORM**
+======================
+
+**About the Project**
+---------------------
+
+The project Interview platform is to provide a digital solution of the interview process. The project. It is meant to ease out the process of hiring and providing efficient way to conduct the process. The project outcome is a web based application which will take into account interviewer and the participants.
```

Comment:
Write the blog from the reader's perspective.

make it write what others can learn from this blog.

it should be step by step, for example, start with requirements, then research, and so on. 

so that anyone reading it can find it indulging.